src/hooks/usePackage.js

```diff
@@ -22,7 +22,7 @@ export const useDasherLatestCoordinates = packageId => {
 export const useGetPackagesIdByCode = packageCode => {
   const [packageInformation, setPackageInformation] = useState({ packages: [] })
   const { loading, error, data } = useQuery(GET_PACKAGE_ID_QUERY_BY_CODE, {
-    variables: { package_code: packageCode },
+    variables: { 'package_code': packageCode },
```

Comment:
Why the single quotes are required? Or is a lint fix?

Reply:
I addressed that issue